c2rust-transpile/src/translator/named_references.rs

```diff
@@ -56,16 +56,16 @@ impl<'c> Translation<'c> {
         reference.and_then(|reference| {
             /// Check if something is a valid Rust lvalue. Inspired by `librustc::ty::expr_is_lval`.
             fn is_lvalue(e: &Box<Expr>) -> bool {
```

Comment:
Can just take an `&Expr` I think.

Reply:
I think we can save this for a more-involved `clippy::borrowed_box` fix that affects a lot of functions at once.